website/guides/advanced/schemas.md.erb

```diff
@@ -0,0 +1,325 @@
+---
+$schema: "/.meta/.schemas/guides.json"
+title: Managing Schemas in Vector
+description: Learn how to manage log schemas with Vector.
+author_github: https://github.com/hoverbear
+tags: ["type: guide", "domain: transforms", "transform: lua"]
+assumptions:
+- You understand the [basic Lua concepts][docs.reference.transforms.lua].
+- You understand the [basic Vector concepts][docs.about.concepts] and understand [how to set up a pipeline][guides.getting-started.your-first-pipeline].
+---
+
+Data comes in all shapes and sizes. Vector has an array (let's call it a vector 😎) of composable functionality for
+decoding your events in the right format, transforming them into the right shape, and passing that data on downstream.
+
+
+## Defining Global Field Names
+
+By default, Vector primarily operates on three fields: `host`, `message`, and `timestamp`.
+
+```json
+{
+  "host": "my.host.com",
+  "message": "<13>Feb 13 20:07:26 74794bfb6795 root[8539]: i am foobar",
+  "timestamp": "2019-11-01T21:15:47+00:00"
+}
+```
+
+It may be that your data does not follow this convention. In this case you can modify the global defaults for all
```

Comment:
```suggestion
All of these fields are set when data is ingested (in a [source][docs.sources]). And it may be that your data does not follow this convention. In this case you can modify the global defaults for all
```

Reply:
Looks good, skipping the unnecessary "and" at the start of the sentence though.